data.js

```diff
@@ -2529,6 +2529,13 @@ module.exports = [
       "PHP",
     ],
   },
+  {
+    name: 'Mike Driscoll',
+    description: 'Python news, beginner to advanced tutorials, interviews and more',
+    url: 'https://www.blog.pythonlibrary.org/',
+    twitter: '@driscollis',
+    tags: ['Python', 'Programming', 'Backend', 'Frontend],
```

Comment:
Hey Mike, please fix the missing quote here 👉 `'Frontend` and change the tag to `Front End` (~~Frontend~~) because it is the most frequent tag.

Reply:
Oops. Sorry about that